recipes/forestopenfermion/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "forestopenfermion" %}
+{% set version = "0.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/forestopenfermion-{{ version }}.tar.gz
+  sha256: a47fd36619ea1cbb973dce33d1716b83e988d8b0acaa97d4bb94ddec7eda50ff
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy >=1.0.0,<2.0.0
+    - openfermion <1.0.0
+    - pyquil >=2.0.0,<3.0.0
+    - python >=3.6
+    - grove <2.0.0
+    - scipy >=1.0.0,<2.0.0
+    - mock
```

Comment:
What's driving the need for this one? I don't see it anywhere upstream.

Reply:
grove requires it and I'm fixing that on an interim basis until it's added upstream: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=420522&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1485

Reply:
Ah, ok.